changelogs/unreleased/gh-9401-anon-replica-synchro.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/replication
```

Comment:
Since this is bugfix, I suppose, we can merge it as is, without `experimental` title

Reply:
Sure.